kalite/topic_tools/__init__.py

```diff
@@ -452,6 +453,8 @@ def get_assessment_item_data(request, assessment_item_id=None):
         assessment_item = main_models.AssessmentItem.objects.get(id=assessment_item_id)
     except main_models.AssessmentItem.DoesNotExist:
         return None
+    except DatabaseError:
+        return None
```

Comment:
I know I'm late to the party here, but the intent is to handle the case when the db doesn't exist, yes? I wonder if `DatabaseError`s are raised in any other circumstances that we might not wish to catch here?